server/api/fetchOpenData/url.get.ts

```diff
@@ -0,0 +1,23 @@
+import { getUrl } from '~/server/prepareInput';
+
+export default defineEventHandler(async (event) => {
+  const feature = getQuery(event).feature as string;
+  if (!feature) {
+    throw createError({
+      statusCode: 400,
+      statusMessage: 'Missing feature',
+    });
+  }
+  console.info('Fetching urls for feature:', feature); // eslint-disable-line no-console
+  try {
+    const urls = await getUrl(feature);
+    return urls;
+  }
+  catch (error) {
+    console.error('Error fetching urls for feature:', feature, error);
+    throw createError({
+      statusCode: 500,
+      statusMessage: 'Error fetching urls',
+    });
+  }
+});
```

Comment:
@alifarooq93 We must extend this feature by also including the following information of each dataset
- organization
- license_title
- license_url

so we should name this endpoint `meta-information.get.ts`